.github/workflows/rust.yml

```diff
@@ -35,4 +36,16 @@ jobs:
         with:
           command: fmt
           args: --all -- --check
+  clippy:
+    name: Clippy
+    runs-on: ubuntu-latest
 
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
+            components: clippy
+      - uses: actions-rs/clippy@master
```

Comment:
@imsnif This needs to be changed asap.

Reply:
Sorry, I'm not sure I understand?

Reply:
The master tag as it's not versioned.
But from what I see it's not really being developed actively